include/reactor-uc/port.h

```diff
@@ -5,23 +5,43 @@
 #include "reactor-uc/reaction.h"
 #include "reactor-uc/reactor.h"
 #include "reactor-uc/trigger.h"
+#include "reactor-uc/environment.h"
 
 typedef struct InputPort InputPort;
 typedef struct OutputPort OutputPort;
 
 struct InputPort {
-  Trigger super;
```

Comment:
The general pattern I went for is to have the super-class be the very first field of the struct such that it can be casted to the super class directly. Is there a good reason for not adhering to this pattern in the case of ports?